skops/card/_markup.py

```diff
@@ -185,7 +189,16 @@ def _code(item: tuple[Any, str]) -> str:
         _, txt = item
         return f"`{txt}`"
 
-    def _table_cols(self, items) -> list[str]:
+    def _table_cols_old(self, items) -> list[str]:  # pragma: no cover
+        # pandoc < 2.5
```

Comment:
any idea when we can drop support for this?

Reply:
I guess as soon as apt updates to a more recent pandoc version. I have no idea when that will be.